packages/terra-site/src/examples/form/examples/FullForm.jsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
```

Comment:
That was a mistake with how I cloned the multi-choice data. I didn't do enough of a clone to prevent this from occurring. It should be fixed now. https://github.com/cerner/terra-core/pull/305/commits/6fa0cb4882422c2a20378ef7ad938c27a571bef1